### ISSUE TITLE
Cannot connect or i2 devices

### ISSUE THREAD

**Issue by sunsetseatime** [eclipse/mosquitto#3112]
Im having an issue connecting mosquito to two devices. One works fine but cant get it to communicate with the other entity.
It currently works fine with solar assistant. It was working fine with Frigate however it doesn't seem to connect with frigate anymore.
Im running home assistant OS with everything up to date.
I cant seem to find any documentation on how to setup mosquito to communicate with more than one entity. This would be very helpful. Thank you

**Comment by ptjm** [comment]
You're not giving a lot of information here. Look for error information in the mosquitto logs. You might want to try running mosquitto in verbose mode (-v flag) while you're debugging the issue.

Taking a wild guess based on the use of "simultaneously" in the topic, make sure that each of your clients is using a unique client ID. User IDs can be shared, but mosquitto will allow only one simultaneous connection for a given client ID.

**Comment by sunsetseatime** [comment]
Thank you for the response. The issue is there is no documentation on how to set this up in home assistant. Perhaps im looking in the wrong place here

**Comment by Daedaluz** [comment]
I'm not sure if this ever was solved, but without more information on what you have tried / trying to do / logs or something I don't  think this issue will move forward.
Could we close or do you want more help?`

**Comment by sunsetseatime** [comment]
> I'm not sure if this ever was solved, but without more information on what you have tried / trying to do / logs or something I don't think this issue will move forward. Could we close or do you want more help?`

I'm having an issue connecting mosquito to two devices. One works fine but cant get it to communicate with the other entity.
It currently works fine with solar assistant. It was working fine with Frigate however it doesn't seem to connect with frigate anymore.
Im running home assistant OS with everything up to date.
I cant seem to find any documentation on how to setup mosquito to communicate with more than one entity. This would be very helpful. Thank you

**Comment by ralight** [comment]
The most common reason for this is that two different clients are connecting with the same client id. I'm afraid I don't know about that in the context of homeassistant though.

**Comment by sunsetseatime** [comment]
> The most common reason for this is that two different clients are connecting with the same client id. I'm afraid I don't know about that in the context of homeassistant though.

Makes sense. I think this is more related to home assistant and how to set it up there. Wonder where  i couls get this sorted
